modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/SnapshotManagerComponent.java

```diff
@@ -196,6 +203,7 @@ protected boolean canSeeRuntimeSnapshot(CoreSession session) {
                                                                                 .stream()
                                                                                 .filter(snap -> !snap.isHidden())
                                                                                 .collect(Collectors.toList());
+        Collections.sort(distribs, getDistributionComparator());
```

Comment:
To move to the stream with help of `sorted` method.

Reply:
same, i'd rather avoid NPE on lambda sort

Reply:
same, I'm not sure we're exposed to lambda NPE as there's no lambda.